src/test/resources/com/conveyal/datatools/gtfs/fake-agency-with-only-calendar-expire-in-2099-with-failed-referential-integrity/stop_times.txt

```diff
@@ -0,0 +1,5 @@
+trip_id,arrival_time,departure_time,stop_id,stop_sequence,stop_headsign,pickup_type,drop_off_type,shape_dist_traveled,timepoint
+only-calendar-trip1,07:00:00,07:00:00,4u6g,1,,0,0,0.0000000,
+only-calendar-trip1,07:01:00,07:01:00,johv,2,,0,0,341.4491961,
+only-calendar-trip2,07:00:00,07:00:00,johv,1,,0,0,0.0000000,
+only-calendar-trip2,07:01:00,07:01:00,4u6g,2,,0,0,341.4491961,
```

Comment:
Can you please rename these tripIDs so that's it's more obvious that this is what causes the referential integrity error?

```suggestion
this-trip-is-not-defined-in-trips-txt,07:00:00,07:00:00,johv,1,,0,0,0.0000000,
this-trip-is-not-defined-in-trips-txt,07:01:00,07:01:00,4u6g,2,,0,0,341.4491961,
```